test/auth.js

```diff
@@ -0,0 +1,51 @@
+let mongoose = require("mongoose");
+let UserModel = require('../dist/models/user.model')
+let chai = require('chai');
+let chaiHttp = require('chai-http');
+let server = require('../dist/server');
+let should = chai.should()
+chai.use(chaiHttp);
+
+describe('Authentication', () => {
+
+    describe('/POST register', () => {
+        it('should register a user', (done) => {
+            let user = {
+                "_id": "12345",
+                "username": "test_user",
+                "email": "testmail@test.de",
+                "password": "test_pass1!",
```

Comment:
In the frontend I also check that the password has at least 1 capitalized letter, I'm not sure we agreed on that but anyway I can remove that feature.

Reply:
thats a good point, i probably have to make all the checks you do again in the backend aswell right?

Reply:
or do i really? Would it be a security flaw if someone bypasses your frontend to create a user with a weak password?

Reply:
Ya, I would say it can't hurt if the backend checks the same parameters again. The backend shouldn't trust anyone anyway, so in my opinion, it should do its own validation.

Reply:
okay then i will create a issue for that.

Reply:
i will update that in the notionpage @domenico. 
but this is unrelated to this pull request so please approve if you dont have any other changes in mind @5h3rr1ll